src/components/IconsRow.tsx

```diff
@@ -108,7 +110,7 @@ export const IconsRow = ({
 	}, [links, visibleChainIndex])
 
 	return (
-		<div className="flex items-center justify-end bg-none overflow-hidden" ref={mainWrapEl}>
+		<div className={`flex items-center justify-${iconsAlignment} bg-none overflow-hidden`} ref={mainWrapEl}>
```

Comment:
this is a bad practice , imagine we don't use justify-end class anywhere in the app , then tailwind doesn't include this className in production build

justify-${iconsAlignment} isn't a valid className, you need to explicitly specific the name 

something like `${iconsAlignment === 'start' ? 'justify-start' : 'justify-end' }` , then tailwind will include both justify-start, justify-end classes in production build

Reply:
Great catch! Didn't think about the treeshaking